### DISCUSSION TITLE
使用 Stylus 定制简悦的样式或主题

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2519] [使用技巧]
> [Stylus](https://chrome.google.com/webstore/detail/clngdbkpkpeebahjckkjfobafhncgmne) 是一款调整网页外观的用户样式管理器，简单的说，它可以给任意页面调整外观，所以也可以定制简悦的阅读模式。

## 社区提供主题目录

- [简洁白](https://github.com/Kenshin/simpread/discussions/2519#discussioncomment-989825)

- [仿 Medium 效果](https://github.com/Kenshin/simpread/discussions/2519#discussioncomment-989827)

- [简约](https://github.com/Kenshin/simpread/discussions/2519#discussioncomment-989828)

- [Dark & Peace](https://github.com/Kenshin/simpread/discussions/2519#discussioncomment-1611395)

- [LXGW WenKai（霞鹜文楷）](https://github.com/Kenshin/simpread/discussions/2519#discussioncomment-2102874)

- [护眼绿](https://github.com/Kenshin/simpread/discussions/2519#discussioncomment-2979208)

## 与简悦自定义样式的区别

简悦本身具有 [自定义样式](http://ksria.com/simpread/docs/#/%E8%87%AA%E5%AE%9A%E4%B9%89%E6%A0%B7%E5%BC%8F) 的功能，Stylus 相对简悦的来说，具有如下优点：

### 方便的 CSS 编辑器

> 具有一个可实时预览的 CSS 编辑器

![image](https://user-images.githubusercontent.com/81074/125186743-b49e2b80-e25e-11eb-836e-1db22eb0af9e.png)

### 可方便切换多个不同的样式

![image](https://user-images.githubusercontent.com/81074/125186783-e7e0ba80-e25e-11eb-9cd3-13253d21118d.png)

### 方便安装

> 点击下面的 Button  自动安装（在已经安装 [Stylus](https://chrome.google.com/webstore/detail/clngdbkpkpeebahjckkjfobafhncgmne) 的前提下）

[![Install simpread white with Stylus](https://img.shields.io/badge/Install%20simpread%20white%20style-Stylus-195bf7.svg)](https://gist.githubusercontent.com/Kenshin/d10d4f4a52c556ef68e073dc443f7e57/raw/simpread_custom_white.user.css)

## 安装

[Chrome Webstore](https://chrome.google.com/webstore/detail/clngdbkpkpeebahjckkjfobafhncgmne)

## 如何定制

这是一个 [现成的模板](https://gist.github.com/Kenshin/d10d4f4a52c556ef68e073dc443f7e57#file-simpread_custom_example-user-css) 内容如下：

```
/* ==UserStyle==
@name           simpread style: xxxx
@namespace      simpread.pro
@version        0.1.0
@description    xxxx style for simpread
@author         xxxx
==/UserStyle== */
@import xxxx
@-moz-document regexp("http[s]://.*")
{
    sr-rd-title {
        
    }

    sr-rd-desc {

    }

    sr-rd-content *, sr-rd-content p, sr-rd-content div {
        
    }
}
```

> 简单的一些说明

![image](https://user-images.githubusercontent.com/81074/125187148-ba950c00-e260-11eb-87fa-8a4934185d92.png)

## 如何提交你的样式

请提交到 https://gist.github.com/Kenshin/d10d4f4a52c556ef68e073dc443f7e57 命名使用务必使用 `impread_custom_xxx.user.css` 的形式。

同时在这个 discussion 下面建立（请替换下面的 `xxx` 为你设置的内容）

```
## 样式的名称

### 安装

[![Install simpread xxx with Stylus](https://img.shields.io/badge/Install%20simpread%20xxx%20style-Stylus-195bf7.svg)](https://gist.githubusercontent.com/Kenshin/d10d4f4a52c556ef68e073dc443f7e57/raw/simpread_custom_xxx.user.css)

### 截图

<请截图>
```
## 关联阅读

- [利用 Google Fonts 为阅读模式添加任何字体](https://github.com/Kenshin/simpread/discussions/1849)

**Comment by Kenshin** [comment]
## 简洁白

> 普通方案 [请访问](https://github.com/Kenshin/simpread/discussions/1849#discussioncomment-987619)

### 安装

[![Install simpread white with Stylus](https://img.shields.io/badge/Install%20simpread%20white%20style-Stylus-195bf7.svg)](https://gist.githubusercontent.com/Kenshin/d10d4f4a52c556ef68e073dc443f7e57/raw/simpread_custom_white.user.css)

### 截图

![image](https://user-images.githubusercontent.com/81074/125187379-f2508380-e261-11eb-85cc-0b4f372b3d28.png)

**Comment by Kenshin** [comment]
## 仿 Medium 效果

> 普通方案 [请访问](https://github.com/Kenshin/simpread/discussions/1849#discussioncomment-987613)

### 安装

[![Install simpread medium with Stylus](https://img.shields.io/badge/Install%20simpread%20medium%20style-Stylus-195bf7.svg)](https://gist.githubusercontent.com/Kenshin/d10d4f4a52c556ef68e073dc443f7e57/raw/simpread_custom_medium.user.css)

### 截图

![image](https://user-images.githubusercontent.com/81074/125187422-2b88f380-e262-11eb-90f5-962f8d7f927a.png)

**Comment by Kenshin** [comment]
## 简约


> 普通方案 [请访问](https://github.com/Kenshin/simpread/discussions/1849#discussioncomment-987615)


### 安装

[![Install simpread simple with Stylus](https://img.shields.io/badge/Install%20simpread%20simple%20style-Stylus-195bf7.svg)](https://gist.githubusercontent.com/Kenshin/d10d4f4a52c556ef68e073dc443f7e57/raw/simpread_custom_simple.user.css)

### 截图

![image](https://user-images.githubusercontent.com/81074/125187451-52472a00-e262-11eb-9cc6-1c5a76c508c2.png)

**Comment by eterlan** [comment]
## Dark & Peace

> 普通方案 [请访问](https://github.com/Kenshin/simpread/discussions/1849#discussioncomment-987615)

### 特点

- 美化了一点排版
- 降低了图片亮度以及字体亮度, 使其比较护眼
- 调整了代码以及链接的字体大小和颜色

### 安装

[![Install simpread Dark & Peace with Stylus](https://img.shields.io/badge/Install%20simpread%20Dakr&Peace%20style-Stylus-195bf7.svg)](https://gist.github.com/Kenshin/d10d4f4a52c556ef68e073dc443f7e57/raw/1dcc0c411297b0369716babebca2b26dcc3762b5/simpread_custom_Dakr&Peace.user.css)

### 截图
![](https://z3.ax1x.com/2021/11/09/It3fPO.jpg)

### 注意
- 推荐使用 Amazon Ember 字体
- 截图中是覆盖了绀鼠主题的效果

**Reply by Kenshin** [reply]
嗨，我明天看下，如果没问题的话，Merge 过来。
然后会发到简悦的各种渠道中。

谢谢付出。🙏

**Reply by Kenshin** [reply]
嗨，此样式已经 Merge 到 https://gist.github.com/Kenshin/d10d4f4a52c556ef68e073dc443f7e57 🙏

另外，建议

> 请详细说明如何使用 **Amazon Ember** 字体

**Reply by Kenshin** [reply]
已发布到 [Telegram Channel](https://t.me/simpreadgroup/37986)

**Comment by Kenshin** [comment]
## LXGW WenKai（霞鹜文楷）

> 普通方案 [请访问](https://github.com/Kenshin/simpread/discussions/1849#discussioncomment-987615)

### 特点

- 使用了 [霞鹜文楷](https://github.com/lxgw/LxgwWenKai) 的字体方案
- 仅改变正文，当遇到 `<code>` 或 `<pre>` 代码段时仍使用默认等宽字体。

### 安装

[![Install simpread LXGW WenKaiwith Stylus](https://img.shields.io/badge/Install%20simpread%20LXGW%20WenKai%20style-Stylus-195bf7.svg)](https://gist.github.com/Kenshin/d10d4f4a52c556ef68e073dc443f7e57/raw/simpread_custom_lxgwwenkai.user.css)

### 截图

![image](https://user-images.githubusercontent.com/81074/152328039-b33bba1e-d557-4bec-b0c1-c6f51ac9366e.png)

### 注意

- 霞鹜文楷 [下载地址](https://github.com/lxgw/LxgwWenKai#%E4%B8%8B%E8%BD%BD%E5%AD%97%E4%BD%93)

**Comment by lyserenity** [comment]
##  护眼绿

### 特点
*  修改了背景色和链接的字体色

### 安装

[![Install simpread Green Eye Stylus](https://img.shields.io/badge/Install%20simpread%20Green%20Eye%20style-Stylus-195bf7.svg)](https://gist.github.com/Kenshin/d10d4f4a52c556ef68e073dc443f7e57/raw/a572c97d01efbbf2ff1b78840a735bc1ccd36a58/simpread_custom_green.user.css)

### 截图

<img width="1341" alt="image" src="https://user-images.githubusercontent.com/32646062/174471330-29a0d093-a18c-4fe4-85bf-988e17d81967.png">

### 版权说明

> 配色来自 [oldj](https://twitter.com/oldj) 兄的作品 [WonderPen](https://wonderpen.app/) 一个主打写作的 App

![image](https://user-images.githubusercontent.com/81074/174472057-1f3aa9e6-06be-4ac8-853f-23d852065b0c.png)

**Comment by john-walks-slow** [comment]
## 柔焦

### 特点

- 鼠标经过某一元素时，将其他元素渐渐虚化、淡化，朦胧而梦幻
- 鼠标停留在页面边距时，取消虚化效果
- 不改变字体、配色。可以和其他主题共同使用

### 安装

[![Install simpread softfocus with Stylus](https://img.shields.io/badge/Install%20simpread%20softfocus%20style-Stylus-195bf7.svg)](https://gist.github.com/john-walks-slow/c6ec0c67bca2d78d2d987f34b2967a11/raw/1e46e60fcf82419391ccec2e2360f8c6d97f7591/simpread_custom_softfocus.user.css)

### 截图
![image](https://github.com/Kenshin/simpread/assets/34511166/d9a92062-ecf5-48e7-8092-897ec102a0e0)
![image](https://github.com/Kenshin/simpread/assets/34511166/d8395689-6f1d-4b51-b6f1-86c8840177cb)

### 视频演示

![2024-03-02_10-54-31 (1)](https://github.com/Kenshin/simpread/assets/81074/338e57d4-0d79-419a-9d80-97c8028a34a8)

**Reply by Kenshin** [reply]
Cool ~ 这个 👍 👍 👍

谢谢分享 ❤️

***

PS：链接失效了 😂

**Reply by john-walks-slow** [reply]
真抱歉！我忘记回复了。

这是正确的链接（我 fork 了用户样式的 gist）：[simpread_custom_softfocus.user.css](https://gist.github.com/john-walks-slow/c6ec0c67bca2d78d2d987f34b2967a11#file-simpread_custom_softfocus-user-css)

[![Install simpread softfocus with Stylus](https://img.shields.io/badge/Install%20simpread%20softfocus%20style-Stylus-195bf7.svg)](https://gist.github.com/john-walks-slow/c6ec0c67bca2d78d2d987f34b2967a11/raw/1e46e60fcf82419391ccec2e2360f8c6d97f7591/simpread_custom_softfocus.user.css)

**Reply by Kenshin** [reply]
哈，小事情。

我帮你更新了使用链接 & 增加了一个 gif 演示。

PS：看了下代码，很有意思的实现方式，思路清奇，效果斐然！

**Reply by john-walks-slow** [reply]
感谢！看小说时的突发奇想哈哈
